code/script.js

```diff
@@ -27,25 +68,97 @@ const showMessage = (message, sender) => {
-  showMessage("Hello there, What's your name?", 'bot');
+  botAnswer(`Hello my friend, What's your name and how can I help you today?`, 'bot')
+  console.log('The bot first question')
   // Just to check it out, change 'bot' to 'user' here 👆
-}
+};
+
+// first user answer 
+const nameAnswer = ((event) => {
+  question = 1
+  event.preventDefault()
+  const name = nameInput.value
+  console.log('The user first answer')
+  userAnswer(`My name is ${name} and I want to know more about the spa packages!`, 'user')
+  nameInput.value = ''
+  setTimeout(() => showSpaOptions(name), 1000)
+});
+
+// second question
+const showSpaOptions = () => {
+  question = 2
+  botAnswer(`Welcome to our spa breaks! We can offer you three kinds of spa packages, what are you interested in more?`, 'bot')
+  chat.innerHTML +=
+    ` <div id="btnOptions" class="options"> 
+     <button class="btn" id= 'birthdayButton' value='Birthday spa breaks' type="click">Birthday spa breaks</button>
+     <button class="btn" id= 'dinnerButton' value='Dinner & Spa breaks' type="click">Dinner & Spa breaks</button>
+     <button class="btn" id= 'boxingButton' value='Boxing Day Spa' type="click">Boxing Day Spa breaks</button>
+     </div>
+     `
+
+  document.getElementById('birthdayButton').addEventListener('click', () => userSpa('Birthday spa breaks'))
+  document.getElementById('dinnerButton').addEventListener('click', () => userSpa('Dinner & Spa breaks'))
+  document.getElementById('boxingButton').addEventListener('click', () => userSpa('Boxing Day Spa breaks'))
+
+};
+
+// third question
+const choosingOptions = (choice) => {
+  question = 3
+  const btnOptions = document.getElementById('btnOptions')
+  btnOptions.remove();
+  console.log("Choice: " + choice);
+  if ((choice === 'Boxing Day Spa breaks')) {
+    botAnswer(`Great! You choose ${choice}! That will cost 2000kr. Could you please write your email address so we can send you the information and the payment method.`, 'bot')
+  } else if (choice === 'Dinner & Spa breaks') {
+    botAnswer(`Perfect! ${choice} will cost 1000kr. Alcoholic drinks are not included. Could you please write your email address so we send you the information and the payment method.`, 'bot')
+  } else {
+    botAnswer(`Nice! ${choice} will cost 1500kr. Could you please write your email address so we send you the information and the payment method.`, 'bot')
+  }
+
+};
+
+
+// fourth question
+const emailAddressAnswer = ((event) => {
+  question = 3
+  const email = nameInput.value
+  console.log('emailTest')
+  userAnswer(`My e-mail adress is ${email}.`, 'user')
+  nameInput.value = ''
+  setTimeout(theEndAnswer, 2000);
+});
+
+
+// fifth question
+const theEndAnswer = () => {
+  question = 4
+  botAnswer(`Perfect! In a few minutes you will receive an email. Enjoy your day with us!`, 'bot')
+  console.log('the end')
+  
+};
```

Comment:
It could maybe be a better user experience if it was more clear that the conversation was done.  Example that the input text bar would disappear, and that the page then would reload?